sencha-workspace/SlateAdmin/app/view/people/details/Courses.js

```diff
@@ -35,6 +35,7 @@ Ext.define('SlateAdmin.view.people.details.Courses', {
     items: {
         xtype: 'grid',
         border: false,
+        columnLines: false,
```

Comment:
@ryon would it make sense for some of these properties to be applied as defaults for all grids by an override in our theme?